src/Player/Player.cpp

```diff
@@ -32,7 +31,7 @@ void Player::init(const al::ActorInitInfo& initInfo) {
 void Player::control() {
     if (al::isPadTriggerL(mPort)) {
         al::setVelocityZero(this);
-        al::setTrans(this, {100.0f, 0.0f, 800.0f});
+        al::setTrans(this, {0.0f, 100.0f, 800.0f});
```

Comment:
Was it wrong before?

Reply:
Yup! It was mismatching with no comment, this fixes it.